packages/wdio-protocols/protocols/webdriver.json

```
@@ -199,6 +199,74 @@
       }
     }
   },
+  "/session/:sessionId/print": {
+    "POST": {
+      "command": "printPage",
+      "description": "The Print Page command renders the document to a paginated PDF document.",
+      "ref": "https://w3c.github.io/webdriver/#print-page",
+      "parameters": [{
+        "name": "orientation",
+        "type": "string",
+        "description": "page orientation. Default: 'portrait'",
+        "required": false
+      }, {
+        "name": "scale",
+        "type": "number",
+        "description": "page scale. Default: 1",
+        "required": false
+      }, {
+        "name": "background",
+        "type": "boolean",
+        "description": "page background. Default: false",
+        "required": false
+      }, {
+        "name": "width",
+        "type": "number",
+        "description": "page width in cm. Default: 21.59 from page",
+        "required": false
+      }, {
+        "name": "height",
+        "type": "number",
+        "description": "page height in cm. Default: 27.94 from page",
+        "required": false
+      }, {
+        "name": "top",
+        "type": "number",
+        "description": "page margin in cm from top margin. Default: 1",
+        "required": false
+      }, {
+        "name": "bottom",
+        "type": "number",
+        "description": "page margin in cm from bottom margin. Default: 1",
+        "required": false
+      }, {
+        "name": "left",
+        "type": "number",
+        "description": "page margin in cm from left margin. Default: 1",
+        "required": false
+      }, {
+        "name": "right",
+        "type": "number",
+        "description": "page margin in cm from right margin. Default: 1",
+        "required": false
+      }, {
+        "name": "shrinkToFit",
+        "type": "boolean",
+        "description": "shrink pdf to fit in page. Default: true",
+        "required": false
+      }, {
+        "name": "pageRanges",
+        "type": "Array<Object>",
```

Our internal parser will only understand this:
```suggestion
        "type": "object[]",
```